### ISSUE TITLE
More Information in README request

### ISSUE THREAD

**Issue by DSMasterson** [dfeich/org8-wikiexporters#6]
Questions I'd like to see answered in the README:

1. Is the TiddlyWiki that is produced a complete TW?  Or do we have to combine it with up-to-date TW Javascript?
2. Is the TiddlyWiki that is produced a pre- or post- 5.0 TW?
3. Are there any other limits on the TW produced?

Thank you

**Comment by DSMasterson** [comment]
Tested this a little.  It seems to produce an outline in TW form, but that (I think) will not be a usable TW without the CSS/Javascript to go with it.  Can you explain in the README how to turn the exported output into a usable TiddlyWiki?

**Comment by dfeich** [comment]
No, this just translates Org syntax into normal TiddlyWiki Syntax, so it's made for copying the exported output into a tiddler entry. So, it's not made for exporting into a standalone tiddlywiki. But you easily could wrap it into such a solution.

I wrote this a long time ago while my knowledge base was kept in Tiddly, but I had already started to move to Emacs Org mode (and never looked back). So, the conversion syntax  based on TiddlyWiki of that aera.

Org mode + export to a static website using org-publish for consuming on mobile browsers - that' what I have been using for the years since then.... But maybe exporting a big org document with pictures to a full standalone TiddlyWiki could still be interesting for some users / use cases.

I should indeed add a few comments in the README, so that people don't stumble over this old package with wrong expectation.  

 . Just took a look for the first time in years on what's going on with Tiddly.. so, still seems to be alive.
